### PR TITLE
Fix missing config schema for pantheon_advanced_page_cache.settings

### PR DESCRIPTION
**Pull request by davereid-pfg**
Resolves https://www.drupal.org/project/pantheon_advanced_page_cache/issues/3194124

### CODE REVIEW

File: config/schema/pantheon_advanced_page_cache.schema.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Schema for the configuration files of the Pantheon Advanced Page Cache module.
+
+pantheon_advanced_page_cache.settings:
+  type: config_entity
```
Comment (clayfreeman) [inline config/schema/pantheon_advanced_page_cache.schema.yml]: This should be `type: mapping` or `type: config_object`, no?

**Review by clayfreeman — CHANGES_REQUESTED** [review]

File: config/schema/pantheon_advanced_page_cache.schema.yml (hunk 1 above)
Comment (davereid-pfg) [reply config/schema/pantheon_advanced_page_cache.schema.yml]: Yup, you're right!

**Review by clayfreeman — APPROVED** [review]

**Comment by dpagini** [comment]
👍 I'd also like to see this move forward and be added to the module.

**Comment by kporras07** [comment]
Looks good. Thanks!

**Comment by kporras07** [comment]
Looks good. Thanks!

**Comment by davereid-pfg** [comment]
Any chance we could possibly get an updated release of the module @kporras07?

**Comment by kporras07** [comment]
@davereid-pfg https://www.drupal.org/project/pantheon_advanced_page_cache/releases/8.x-1.3 just created. Thanks!

**Pull request merged**